templates/irc_bridge_production.yaml

```diff
@@ -0,0 +1,141 @@
+      sendConnectionMessages: true
+      quitDebounce:
+        enabled: false
+        quitsPerSecond: 5
+        delayMinMs: 3600000 # 1h
+        delayMaxMs: 7200000 # 2h
+      modePowerMap:
+        o: 50
+        v: 0
+      botConfig:
+        enabled: true
+        nick: "UbuntuLiberaBot"
+        username: "ubunbtuliberabot"
+        joinChannelsIfNoUsers: true
+      privateMessages:
+        enabled: true
+        federate: true
+      dynamicChannels:
+        enabled: true
+        createAlias: true
+        published: false
+        useHomeserverDirectory: true
+        joinRule: public
+        federate: true
+        aliasTemplate: "#libera_$CHANNEL"
+      membershipLists:
+        enabled: true
+        floodDelayMs: 10000
+        global:
+          ircToMatrix:
+            initial: false
+            incremental: true
+            requireMatrixJoined: false
+          matrixToIrc:
+            initial: false
+            incremental: true
+        ignoreIdleUsersOnStartup:
+          enabled: false
+          idleForHours: 720
+          exclude: "foobar"
+      matrixClients:
+        userTemplate: "@libera_$NICK"
+        displayName: "$NICK"
+        joinAttempts: -1
+      ircClients:
+        nickTemplate: "$DISPLAY[m]"
+        allowNickChanges: true
+        maxClients: 30
+        ipv6:
+          only: false
+        idleTimeout: 0
+        reconnectIntervalMs: 5000
+        concurrentReconnectLimit: 50
+        lineLimit: 3
+        realnameFormat: "mxid"
+        kickOn:
+          channelJoinFailure: true
+          ircConnectionFailure: true
+          userQuit: true
+  bridgeInfoState:
+    enabled: false
+    initial: false
+  ident:
+    enabled: false
+    port: 1113
+    address: "::"
+  logging:
+    level: "debug"
+    logfile: "debug.log"
+    errfile: "errors.log"
+    toConsole: true
+    maxFiles: 5
+  metrics:
+    enabled: false
+    port: 7001
+    host: 127.0.0.1
+    userActivityThresholdHours: 72 # 3 days
+    remoteUserAgeBuckets:
+      - "1h"
+      - "1d"
+      - "1w"
+  debugApi:
+    enabled: false
+    port: 11100
+  provisioning:
+    enabled: false
+    widget: false
+    requestTimeoutSeconds: 300
+    rules:
+      userIds:
+        exempt:
+          - "@doubleagent:example.com"
+        conflict:
+          - "@.*:example.com"
+    roomLimit: 50
+    http:
+      port: 7700
+      host: 127.0.0.1
+  passwordEncryptionKeyPath: "/data/config/passkey.pem"
+  matrixHandler:
+    eventCacheSize: 4096
+    shortReplyTemplate: "$NICK: $REPLY"
+    longReplyTemplate: "<$NICK> \"$ORIGINAL\" <- $REPLY"
+    shortReplyTresholdSeconds: 300
+  userActivity:
+    minUserActiveDays: 1
+    inactiveAfterDays: 30
+  ircHandler:
+    mapIrcMentionsToMatrix: "on" # This can be "on", "off", "force-off".
+    powerLevelGracePeriod: 1000
+  permissions:
+    '@merkata:ubuntu.com': admin
+    '@amandahla:ubuntu.com': admin
+    '@kewisch:ubuntu.com': admin
+    '@ilvipero:ubuntu.com': admin
```

Comment:
If we're making a charm for everyone to use, this shouldn't be hardcoded

Reply:
I agree